src/api/push.rs

```diff
@@ -185,13 +183,12 @@ pub async fn push_logout(user: &User, acting_device_uuid: Option<String>, conn:
                 "UserId": user.uuid,
                 "Date": user.updated_at
             }
-        });
-        send_to_push_relay(data).await;
+        })));
```

Comment:
This can be shorter and easier to read like this i think.
Also, we probably always want to send a logout request i think, but if that is not the case, then the `if Device::check...` check can be added of course.


```suggestion
    let acting_device_uuid: Value = acting_device_uuid.map(|v| v.into()).unwrap_or_else(|| Value::Null);
    // Always send a logout command, doesn't matter if a user has a push enabled device or not.
    tokio::task::spawn(send_to_push_relay(json!({
        "userId": user.uuid,
        "organizationId": (),
        "deviceId": acting_device_uuid,
        "identifier": acting_device_uuid,
        "type": UpdateType::LogOut as i32,
        "payload": {
            "userId": user.uuid,
            "date": user.updated_at
        }
    })));
```

Reply:
Done.